### PR TITLE
Update satellite and pcap for AWS

### PR DESCRIPTION
**Pull request by abhishek-levo**

### CODE REVIEW

File: docs/api-observability/install-guide/install-sensor/pcap-sensor.md [hunk 1]
```diff
@@ -12,22 +12,21 @@ sidebar_position: 3
...
-
  - [Install via Docker](#install-docker)
  - [Install on Kuberenetes](#install-kubernetes)
 
 <a id="install-fargate"></a>
 
 ## Install Sensor on Fargate
 
-<a id="install-fargate-using-json"></a>
+The pcap Sensor can be installed as a sidecar on a already present AWS task, and can be added to its task definition via the AWS Console.
```
Comment (VamsiNerella) [inline docs/api-observability/install-guide/install-sensor/pcap-sensor.md:27]:
```suggestion
The pcap Sensor can be installed as a sidecar on an existing AWS task by adding to it's task definition via the AWS Console.
```

File: docs/api-observability/install-guide/install-satellite.mdx [hunk 2]
```diff
@@ -306,3 +309,224 @@ The Collector now runs in a container, and is reachable on the host via port 431
...
+metadata:
+  name: ${CLUSTER_NAME}
+  region: ${AWS_REGION}
+
+vpc:
+  subnets:
+    private:
+      us-west-2a: { id: subnet-0d09e999a579e86ba }
+      us-west-2b: { id: subnet-038ae44660fbc358b }
+      us-west-2c: { id: subnet-06f1946a447e569d1 }
```
Comment (VamsiNerella) [inline docs/api-observability/install-guide/install-satellite.mdx:353]: we should add a note to change this if they want to install in different region.

File: docs/api-observability/install-guide/install-satellite.mdx [hunk 3]
```diff
@@ -306,3 +312,527 @@ The Collector now runs in a container, and is reachable on the host via port 431
...
+}
+```
+
+### 2. Run the Satellite
+
+Now that we have created a task definition, we will start our satellite up as a service in an ECS cluster.
+
+---- to add ----
+
+### 3. ---- to add -----
```
Comment (ricekot) [inline docs/api-observability/install-guide/install-satellite.mdx:814]: Please update this

Comment (abhishek-levo) [reply docs/api-observability/install-guide/install-satellite.mdx]: Addressed in #151 [72ab011](https://github.com/levoai/docs/pull/151/commits/72ab011739bbcfa861f8b6be35e17c10c879c4b5)

**Pull request merged**